app/serializers/v1/user_intervention_serializer.rb

```diff
@@ -17,7 +17,15 @@ class V1::UserInterventionSerializer < V1Serializer
   end
 
   attribute :blocked do |object|
-    object.intervention.shared_to_invited? && object.intervention.intervention_accesses.pluck(:email).map(&:downcase).exclude?(object.user.email)
+    if object.intervention.shared_to_invited?
+      object.intervention.intervention_accesses
+            .pluck(:email)
+            .map(&:downcase)
+            .exclude?(object.user.email) &&
+        object.user.roles.exclude?('predefined_participant')
+    else
+      false
+    end
```

Comment:
Maybe we can do something like that:
```suggestion
   object.intervention.shared_to_invited? &&
      object.intervention.intervention_accesses
            .pluck(:email)
            .map(&:downcase)
            .exclude?(object.user.email) &&
        object.user.roles.exclude?('predefined_participant')
```